src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -75,7 +72,7 @@ private void openWindow(Stage mainStage) {
         GUIGlobals.init();
```

Comment:
Some lines earlier you have JabRefGui.getMainFrame() instead of mainFrame

Reply:
Yes, it was incosistent, i used only `mainFrame` now in the whole class.